code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/github/resource/GithubClient.java

```diff
@@ -318,4 +330,27 @@ public GithubAccountInfo getUserOrgById(String id) {
     public String getName() {
         return GithubConstants.GITHUB_CLIENT;
     }
+
+    public Set<Identity> getIdentities(String accessToken) {
```

Comment:
This was added so that there is a clearer pattern seen in the token creator classes.